src/main/java/edu/wisc/my/rssToJson/service/RsstoJsonServiceImpl.java

```diff
@@ -20,6 +23,17 @@
 
     private RssToJsonDao rssToJsonDao;
 
+    // JsonArray of unicode characters
+    // offender is the unicode character we're replacing
+    // replacement is the ASCII character we're subbing in.
+    private String unicodeToClean = "{\"replacements\":[{" +
```

Comment:
I'd have named this something more like "characterMappings". This String isn't going to be itself cleaned so the "toClean" isn't doing it for me -- I don't think of my mop as my "mopToClean" either. :)